front/src/form/bsda/stepper/steps/Destination.tsx

```diff
@@ -7,7 +7,7 @@ import { getInitialCompany } from "form/bsdd/utils/initial-state";
 export function Destination({ disabled }) {
   const { values, setFieldValue } = useFormikContext<Bsda>();
   const hasNextDestination = Boolean(
-    values.destination?.operation?.nextDestination
+    values.destination?.operation?.nextDestination?.company
```

Comment:
```suggestion
    values.destination?.operation?.nextDestination?.company?.siret
```
Est-ce qu'on ne devrait pas carrément aller chercher le siret ici ? 
est-ce qu'on ne devrait pas aller chercher

Reply:
Le problème, c'est qu'au moment ou je dis "je veux une nextDestination", j'ai un siret qui est une string vide. Idem si je décoche l'entreprise dans le sélecteur.
Si on decend au niveau de la company on est obligé d'introduire une autre variable pour le moment ou on clique sur la coche.